src/lib/djerba/plugins/fusion/fusion_template.html

```diff
@@ -1,24 +1,37 @@
 <%
   import djerba.core.constants as core_constants
-  from djerba.plugins.fusion.plugin import main as fusion
+  import djerba.plugins.fusion.constants as fusion
   from djerba.plugins.fusion.html import make_table_rows
+  from djerba.util.html import html_builder
+
 %>
 
+${html_builder().section_cells_begin("<h2>FUSIONS AND STRUCTURAL VARIANTS</h2>","main")}
 
-<hr class="big-white-line" ><div class="twocellmain"><div class="oneoftwocellmain"><h2>Fusions and Structural Variants</h2></div><div class="twooftwocellmain" ><hr class="big-line" >
+  <style type="text/css">
+    .oncokb-levelP {background: #000000;}
+  </style>
 
-    <p><strong>${results.get(fusion.TOTAL_VARIANTS)}</strong> cancer gene(s) were subject to rearrangement,
-      of which <strong>${results.get(fusion.CLINICALLY_RELEVANT_VARIANTS)}</strong> corresponded to oncogenic fusions, as defined by OncoKB.</p>
+    <p><strong>${results.get(fusion.TOTAL_VARIANTS)}</strong> cancer gene(s) were subject to rearrangement. 
+      <strong>${results.get(fusion.CLINICALLY_RELEVANT_VARIANTS)}</strong> fusion(s) were oncogenic according to OncoKB. 
+      <!--,in addition to which <strong>${results.get(fusion.NCCN_RELEVANT_VARIANTS)}</strong> rearrangement(s) appeared in NCCN's biomarker compendium.</p>-->
```

Comment:
Do we want to keep this comment?
Note that in Mako, you can prefix a comment with `##` so it appears in the template but not the HTML output

Reply:
good to know about the mako commenting
this commenting is just to turn off this sentence until we've approved NCCN markers, which should be soon